### PR TITLE
Removed spell Levitate from C++

### PR DESCRIPTION
**Pull request by raymondtfr**
Made this new PR as requested by @ranisalt in #2081 .

Should the **actual** removing of the spell levitate be here? or just the function loading part (as you indeed told in #2081 ?), I think I have mislead this.

### CODE REVIEW

**Comment by djarek** [comment]
@raymondtfr 
I think that this PR should remove all the dead code related to the Levitate spell, so it's fine as is.

**Comment by ranisalt** [comment]
I meant you should keep the code for the `loadFunction` function just issuing a warning (i.e. remove the last if but keep the rest).

File: src/spells.cpp [hunk 1]
```diff
@@ -884,68 +884,6 @@ bool InstantSpell::configureEvent(const pugi::xml_node& node)
...
-}
-
-}
-
-bool InstantSpell::loadFunction(const pugi::xml_attribute& attr)
-{
-	const char* functionName = attr.as_string();
-	if (strcasecmp(functionName, "levitate") == 0) {
-		function = Levitate;
-	} else {
```
Comment (ranisalt) [inline src/spells.cpp]: What I said is to keep this function here, remove the if and just keep the else branch here:

```cpp
bool InstantSpell::loadFunction(const pugi::xml_attribute& attr)
{
	const char* functionName = attr.as_string();
	std::cout << "[Warning - InstantSpell::loadFunction] Function \"" << functionName << "\" does not exist." << std::endl;
	return false;
}
```

The rest is ok and you can add to the other PR.

File: src/spells.cpp [hunk 2]
```diff
@@ -1140,8 +1078,6 @@ bool InstantSpell::internalCastSpell(Creature* creature, const LuaVariant& var)
 {
```
Comment (ranisalt) [inline src/spells.cpp]: The changes below here can also be undone.

File: src/spells.cpp (hunk 1 above)
Comment (raymondtfr) [inline src/spells.cpp]: Even though it doesn't have any function now (when Levitate is moved to Lua.)?

**Comment by WibbenZ** [comment]
Yes for ppl who havent updated.
But IMO print that the functions have been moved to Lua or smth, if we for some reason add C++ functions again we can edit the string.

File: src/spells.cpp (hunk 1 above)
Comment (ranisalt) [inline src/spells.cpp]: Yes, then you can open a PR removing leftovers if you want, or I can do it as soon as I merge.

**Comment by ranisalt** [comment]
I don't think it will be moved back to C++ ever, but we can overload the `loadFunction` in the base `Event` class to issue a warning.

File: src/spells.cpp (hunk 1 above)
Comment (raymondtfr) [inline src/spells.cpp]: Ok, then I updated the other PR with removing only the function Levitate enclosed in a namespace.

And here I'll just do what you suggested.

**Comment by ranisalt** [comment]
As both PRs and a review was pending weren't compiling I went ahead and commited to master with fixes.

**Pull request closed without merging**